tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
```

Comment:
`conftest.py` is a special filename recognized by pytest and is run before any tests are executed. Typically contains various utility functions and "test fixtures".